pkg/operation/botanist/controlplane.go

```diff
@@ -708,17 +708,75 @@ func (b *Botanist) DeployControlPlane(ctx context.Context) error {
 	})
 }
 
+const controlPlaneExposureSuffix = "-exposure"
+
+// DeployCloudSpecificControlPlane creates the `ControlPlane` extension resource with purpose `exposure` in the shoot
+// namespace in the seed cluster. Gardener waits until an external controller did reconcile the cluster successfully.
+func (b *Botanist) DeployCloudSpecificControlPlane(ctx context.Context) error {
```

Comment:
```suggestion
func (b *Botanist) DeployControlPlaneExposure(ctx context.Context) error {
```

(adapt other names accordingly, please)